pkg/api/types.go

```diff
@@ -1391,8 +1391,34 @@ type VolumeMount struct {
 	// Defaults to "" (volume's root).
 	// +optional
 	SubPath string
+	// mountPropagation is the mode how are mounts in the volume propagated from
```

Comment:
nit: nit: "mountPropagation determines how mounts are propagated from the host to container and the other way around. When not set, defaults to MountPropagationHostToContainer."

Reply:
@bgrant0607 was against using world "default" when something does not fill the default in `default.go`. https://github.com/kubernetes/kubernetes/pull/19921#issuecomment-176575106

Reply:
Correct. Is there a reason we wouldn't fill in the default?